src/main/java/codewars/tasks/java/Block.java

```diff
@@ -0,0 +1,40 @@
+package codewars.tasks.java;
+
+public class Block {
+
+        private int width;
+        private int length;
+        private int height;
+
+        Block(int[] measurements) {
+            if(measurements.length == 3) {
+                this.width = measurements[0];
+                this.length = measurements[1];
+                this.height = measurements[2];
+            }
+        }
+
+        public int getWidth() {
+            return width;
+        }
+
+        public int getLength() {
+            return length;
+        }
+
+        public int getHeight() {
+            return height;
+        }
+
+        public int getVolume() {
+            int volume = getWidth()*getLength()*getHeight();
+            System.out.println("Block volume with params "+length+" "+width+" "+height+" is "+volume);
```

Comment:
Better use String.format for a string with several params:
`String.format("Block volume with params: l:%d w:%d h:%d v:%d", this.width * this.length * this.height, volume)`